src/idl/src/annotation.c

```diff
@@ -722,6 +1015,67 @@ annotate_position(
   return IDL_RETCODE_OK;
 }
 
+static idl_retcode_t
+annotate_try_construct(
+  idl_pstate_t *pstate,
+  idl_annotation_appl_t *annotation_appl,
+  idl_node_t *node)
+{
+  assert(annotation_appl);
+
+  const char *str = NULL;
+  const idl_type_spec_t *ts = NULL;
+  const idl_annotation_appl_t **annotation_appl_p = NULL;
+  idl_try_construct_t *try_construct_p = NULL;
+
```

Comment:
If the min/max annotations need to check for the field having been annotated before already, shouldn't this one do so, too?

Reply:
Same as the previous comment.